libevm/VM.cpp

```diff
@@ -1236,6 +1236,7 @@ void VM::interpretCases()
 
 		CASE(RETURNDATACOPY)
 		{
+			ON_OP();
```

Comment:
Is this change related to the "inserting dot" bug?

Reply:
@holiman  suggested adding this into this PR. it is related to fixing debug output

Reply:
@cdetrio  should test it

Reply:
I'll test it

Reply:
Would this change also affect Remix?  @yann300 ?

Reply:
That will change the VM trace output yes but I think this change does not require to update Remix